EssentialFeed/EssentialFeed/Image Comments/API/ImageCommentsEndpoint.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public enum ImageCommentsEndpoint {
+	case get(imageID: String)
+
+	public func url(baseURL: URL) -> URL {
+		switch self {
+		case .get(let imageID):
+			return URL(string: baseURL.absoluteString.appending("/v1/image/\(imageID)/comments"))!
```

Comment:
This logic can be simpler and eliminate force unwrapping by using the URL `appendingPathComponent` method:

```suggestion
			return baseURL.appendingPathComponent("/v1/image/\(imageID)/comments")
```